webpack.config.js

```diff
@@ -18,7 +18,7 @@ const defaultEntry = `./${project}/index.js`;
 const ENTRIES = {
   tutor: {
     tutor: defaultEntry,
-
+    lmspair: './tutor/lms-pair.js',
```

Comment:
out of curiosity, why is this a new entry?

Reply:
This sets up a new build that's loaded by the BE for pairing a course.  it's a specialized screen that doesn't allow navigation.  Because of that we can jettison quite a bit of the existing code so it loads faster.